File: Greedy/Cpp/1120.cpp
// 문자열
#include <iostream>
#include <string>
using namespace std;

int main() {
    string a, b;
    cin >> a >> b;
    if (a.length() < b.length()) swap(a, b);
    int la = a.length();
    int lb = b.length();
    int min = 1000000;
    for (int i = 0; i <= la - lb; i++) {
        int wrong = 0;
        for (int j = i; j < i + lb; j++) {
            if (a[j] != b[j - i]) wrong++;
        }
        if (wrong < min) min = wrong;
    }
    cout << min << '\n';
}

File: Greedy/Python/1120.py
#  문자열
a, b = input().split()
la = len(a)
lb = len(b)
if la < lb:
    a, b = b, a
    la, lb = lb, la
min = 100000
for i in range(la-lb + 1):
    wrong = 0
    for j in range(i, i+lb):
        if a[j] != b[j-i]:
            wrong += 1
    if min > wrong:
        min = wrong
print(min)
